apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineApplication.java

```diff
@@ -304,16 +310,18 @@ private void createJobsAndRunPipeline(
       MetricRegistry appMetrics,
       HikariDataSource pooledDataSource)
       throws FatalAppException {
+    final var clock = Clock.systemUTC();
+
```

Comment:
`Clock.systemUTC()` was being used directly in several places.  Using this single reference is simpler and still allows us to pass specific clocks into these methods in tests.